### ISSUE TITLE
When initiating a voice call, sound is quiet on recipient iOS device

### ISSUE THREAD

**Issue by spagy** [signalapp/Signal-Android#11336]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When I initiate a video call to someone on iOS my voice is quite for them. (Tested with two different iOS devices).
My voice is at a normal/expected volume when I receive a video call from someone on iOS. (Tested with the same two iOS devices).
This is not an issue when calling another Android device. (Tested with one other Android device)
I'm not sure whether this is an issue with the Android client or the iOS client; I'd be happy to submit an issue for the iOS client if that seems to be where the issue is.

### Steps to reproduce
- Start a voice call to someone on iOS

**Actual result:** They complain about my voice being quiet. I suggest they call me back.
**Expected result:** They can hear me easily.

### Device info

**Device:** Motorola Moto G6
**Android version:** 11
**Signal version:** 5.11.5

### Link to debug log
https://debuglogs.org/18271073d1e19580620311cba1fd6f6c396164c3692cd5038d707dc14165b047

**Comment by pricead** [comment]
I'm experiencing a similar issue when calling from an Android (Samsung S20 FE) to an iPhone 11 Pro. The iPhone user says my audio is very low and difficult to hear. We don't experience this issue when using other video chat apps, such as Duo.

**Comment by SlabofPork** [comment]
I had this problem on a Samsung Galaxy 3, as well. The iOS app seems to have a bug where when a video call begins (I checked the Signal iOS support forum), the iPhone still thinks it's in media mode, and as such does not activate the speaker. The iPhone user has to mute, and then unmute their phone in the video call, and that should trigger the iPhone to understand that this is indeed a phone call, and the speaker should be on. I tested it by calling a family member (I on the Galaxy, her on an iPhone), and I ran into the same problem as you described. She then muted and then unmuted her phone, and the problem was fixed. 

TL;DR: Have the iPhone user mute, then unmute the call in app. They should now be able to hear you.

**Comment by yousefamar** [comment]
This happens on my Pixel 4 -- when I call my mum (iPhone) she says my voice is way too quiet, but it doesn't happen when we hang up and she calls me back

**Comment by cody-signal** [comment]
Thanks for the reports y'all. We've let our iOS and Calling teams know and they will hopefully resolve it soon.

**Comment by spagy** [comment]
I have recently had this issue but in reverse, I can't remember whether I initiated the call or not, but the other party was on iOS and I was on android.

I could hear them at the volume I would expect a (non loudspeaker) voice call to be but the call was a video call. I fixed this by disabling my camera (to switch to a voice call) and then re-enabling the camera.

Doing the same thing seems to work on iOS too.

**Comment by punyidea** [comment]
I have this issue (or a similar one) as well. I believe that my phone is using a different microphone than the one that is intended, when this happens.

**Comment by spagy** [comment]
i'm a bit late for the stale bot but this is still relevant! happy to submit more logs if needed.

**Comment by spagy** [comment]
i've just had this issue (in reverse) again.

i made a video call to an iOS user twice and both times they sounded very quiet to me.

when they called me there was no issue, the volume was as expected.

for all three of these calls, i was using bluetooth headphones. when i've had this issue before it's been without headphones.

@cody-signal would you be able to re-open this issue? or should i create a new issue?

**Comment by nicholas-signal** [comment]
> i've just had this issue (in reverse) again.

If you can submit debuglogs from both sides of the conversation, you should submit a new GitHub Issue and the Calling team can look into it.
